assets/css/content/functions.css

```
@@ -62,6 +57,10 @@
   }
 }
 
+.content-inner .detail-header .icon-action {
+  margin-top: 4px;
```

```suggestion
  margin-top: 3px;
```

This makes the alignment the same as main. Apologies for pixel shaving. :D